src/core/server/ui_settings/saved_objects/migrations.ts

```diff
@@ -75,4 +75,26 @@ export const migrations = {
     }),
     references: doc.references || [],
   }),
+  '8.0.0': (doc: SavedObjectUnsanitizedDoc<any>): SavedObjectSanitizedDoc<any> => ({
+    ...doc,
+    ...(doc.attributes && {
+      attributes: Object.keys(doc.attributes).reduce(
+        (acc, key) =>
+          [
+            'visualization:regionmap:showWarnings',
```

Comment:
could you add a comment about the team owner? We will move this logic to plugins later.

Reply:
How should this comment be structured? is `// owner: Team:Geo`?

Reply:
yes